.travis.yml

```diff
@@ -25,6 +25,16 @@ matrix:
     - name: "ARCH=x86_64 REPO=linux-next"
       env: ARCH=x86_64 REPO=linux-next
       if: type = cron
+    # stable
+    # TODO: make cron only
+    - name: "ARCH=arm64 REPO=4-18"
+      env: ARCH=arm64 REPO=4-18
+    - name: "ARCH=arm64 REPO=4-14"
+      env: ARCH=arm64 REPO=4-14
+    - name: "ARCH=arm64 REPO=4-9"
```

Comment:
Should 4.4 and 4.9 be added just yet since support is not upstream yet?

Reply:
I guess it's a question of having things wired up, even if they're red on day one.  Depends on what we want to do if/when a target goes red; do we want to disable it, or let every cron be red until a fix lands?

I'd prefer to have some automated testing so we can collect warnings from the logs. Maybe we can settle the question about regressions later.

Reply:
I'd argue we should keep everything green. Since 4.14 and 4.19 _should_ work fine and the case block below is super easy to extend with new version (and 4.4 and 4.9 can even remain), I suggest we keep 4.4 and 4.9 off of the matrix until the core Clang support is added to those branches upstream. Alternatively, we could create a `linux-stable` repo on our organization (or branches on the main `linux` repo) and have the backports there until integration upstream just for early testing.

Reply:
Ok, I've updated the patch set.  I want at least one run presubmit, just so we have logs we can point to from our issue tracker or whatever to keep track of what needs to get backported.  Will comment out the configs once we get a run.